76 ips ehds2 optimized (#77)
IHE/ITI.DeIdHandbook: input/pagecontent/ips-ehds-example.md

### Purpose and Overview

> This exemplar demonstrates how the IHE De-Identification Handbook’s process framework is applied to the EHDS2 IPS + VRDR secondary-use scenario. It serves as a concise de-identification profile: defining purpose, recipients, multi-stage process, risk thresholds, and element-by-element treatment to preserve research utility while protecting privacy. See the methodology overview in [Process](process.html).

Reviewer: Replace with:
This exemplar demonstrates how the IHE De-Identification Handbook’s process framework is applied a fictitious epidemiologic example that conforms to the secondary use requirements defined by the EHDS2. This example leverages the International Patient Summary (IPS) and the Vital Records Death Reporting (VRDR) profiles for conveying study data content for this secondary-use scenario. It serves as a concise de-identification profile: defining purpose, recipients, multi-stage process, risk thresholds, and element-by-element treatment to preserve research utility while protecting privacy. See the methodology overview in [Process](process.html).

Author: replace the term "de-identification profile" with "de-identification example"

### Context Analysis

> #### Purpose of collecting data

Author: there is typo

> The permitted purpose is Public Interest research in Public/Occupational Health to study co-morbidities, mortality, occupational risks, and treatment effects. The dataset must be minimized and de-identified to preserve utility (longitudinal trends, outcomes, medication impacts) while controlling re-identification risk per permit conditions.

Reviewer: Replace the first part of sentence with: The permitted purpose for this example

Author: done

- developing products or services that may harm individuals, public health or society at large, such as illicit drugs, alcoholic beverages, tobacco and nicotine products, weaponry or products or services which are designed or modified in such a way that they create addiction, contravene public order or cause a risk for human health;
- carrying out activities in conflict with ethical provisions laid down in national law.

#### Data recipients

- Researchers granted a Data Permit under HDAB oversight
- HDAB staff and Intermediation entity performing de-identification and release
- Authorized analytics environments (controlled access; data use agreements)

#### Data flow

This data flow implements the methodology’s end-to-end analysis described in [Process → Analyze the context → Data flow](process.html). It maps: source clinical systems → HDAB Intermediation (Stage 2) → controlled recipients, and highlights multi-stage requirements, environments, and constraints.

##### end-to-end data flow

> - Before the de-identification processing, EHDS 2 regulations specify that the Health Data User conducts data discovery to identify data that is available for the study, and that an application for a data permit be submitted and approved. The Data Discovery leveraging the HealthDCAT Application Profile (HealthDCAT-AP) determines that the information is available within the HDAB Information Resources, indicating data is available in a standard International Patient Summary (IPS), and mortality data in the Vital Records Death Reporting (IHE VRDR) standard formats as indicated in the conformsTo attribute (see https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-5/).

Reviewer: Replace with: - Before the de-identification processing, EHDS 2 regulations specify that the Health Data User conducts data discovery to identify data that is available for the study, and that an application for a data permit be submitted and approved. The Data Discovery leveraging the HealthDCAT Application Profile (HealthDCAT-AP), a specification developed by the European Commission DG SANTE to fulfill the requirements of Article 77 of Regulation (EU) 2025/327 [[OJ L, 2025/327, 5.3.2025](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-5/#bib-oj%20l,%202025/327,%205.3.2025)] establishing the European Health Data Space (EHDS). This determines that the information is available within the HDAB Information Resources, indicating data is available in a standard International Patient Summary (IPS), and mortality data in the Vital Records Death Reporting (IHE VRDR) standard formats as indicated in the conformsTo attribute (see https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-5/).

Author: done

> - The user submits a Data Access Permit application (drawn from example at: https://data-access.dsa.ec.europa.eu/public/hta/data-access) requests data access authorization providing [in the Research project-specific information](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-5/) section:

Reviewer: Replace with: 
The user submits a Data Access Permit application (drawn from example at: https://data-access.dsa.ec.europa.eu/public/hta/data-access), and requests data access authorization providing [in the Research project-specific information](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-5/) section:

Author: done

- Vital records: VRDR mortality data from civil registries, limited to attributes necessary for surveillance (e.g., death date, cause categories, age bands, residence generalization).
- Agreements: Source data use agreements and HDAB permit terms define allowable elements, pre-release minimization, and any source-side pseudonymization.

##### Data Environments

- Landing zone: Secure HDAB intake environment receiving permitted data from sources; access is restricted and audited.
- Processing zone: HDAB Intermediation environment applying approved minimization, generalization, pseudonymization/anonymization, and outlier suppression according to the permit and risk model.
- Analysis zone: Authorized recipient environments (controlled access or enclave) where researchers analyze de-identified data under data use agreements; outbound sharing is prohibited.
- Transfer controls: Encrypted transport, authenticated endpoints, and governance monitoring across zones to manage context risk.

##### Multi-stage Requirements

> - Stage 1 (at sources): Basic minimization and, where applicable, reversible pseudonymization to safely transfer data to HDAB; direct identifiers removed or replaced per local capability.

Reviewer: At sources OR HDAB that has access to source data.
Update to reflect that this may all be accomplished with the HDAB or at the data source.

Author: replaced with: - Stage 1 (at sources or by HDAB with source access): Basic minimization and, where applicable, reversible pseudonymization to safely transfer data to HDAB; direct identifiers removed or replaced per source/HDAB capability.

##### Regulatory Constraints

- The HDAB data access application management process is primarily set out in Articles 67–69 of the EHDS regulation, which define the procedural context that any organisational or technical solution for secondary use data applications must align with. Articles 67 and 69 includes the requirements for the common application forms for data access applications and data requests, respectively, to be used by applicants, and which provides the essential information for the processing of applications. Article 68 governs the issuance of data permits and the associated obligations of HDABs, whereas Article 69 provides similar provisions for data requests.
- The secondary use of electronic health data is based on pseudonymised or anonymised data, in order to preclude the identification of the data subjects (Recital 53 in the EHDS regulation)
- Data anonymisation, pseudonymisation, and linkage techniques are addressed in M7.2 Technical specification for Health Data Access Bodies on data minimisation and de-identification, and M7.5 Guideline for Health Data Access Bodies on linkage of health datasets.

### Data assessment

#### Data content

> Dataset is primarily structured clinical data from IPS (FHIR R4) and mortality attributes from VRDR. It is longitudinal (multiple records per subject across time), minimizes free text (procedure descriptions removed), excludes imaging, and contains no binary device logs. Semi-structured risks (filenames, metadata) are minimized by standardized export and HDAB processing.

Reviewer: Change to: The Dataset is primarily structured clinical data from IPS (FHIR R4) and mortality attributes from VRDR (FHIR R4). It is longitudinal (multiple records per subject across time), minimizes free text (procedure descriptions removed), excludes imaging, and contains no binary device logs. Semi-structured risks (filenames, metadata) are minimized by standardized export and HDAB processing.

Author: done

Policy for secondary use includes data minimisation for the use of secondary health data. This includes limiting the amount, type, and granularity of data during data preparation. The application for data access includes a request for the following data and de-identification methods:

##### Patient

- Pseudonymized name

> - Patient location is important to the study. Address locations generalized to the initial 3-digits of the postal code for dataminimization

Author: ```suggestion
- Patient location is important to the study. Address locations generalized to the initial 3-digits of the postal code for data minimization
```

  - Occupation either usual occupation or current occupation is needed to identify incidents and to help identify risk factors associated with identified occupations
  - Industry either usual industry or current industry is needed to identify incidents and to help identify risk factors associated with identified occupations

- Occupation and Industry will be reviewed for outliers and suppressed
- All other attributes associated with Social History are removed for data minimization
- Pregnancy History
  - Pregnancy status pregnancy information is needed for the study to review potential impact of the incident on pregnancy
  - Estimated Delivery date is needed relative to incident and treatment dates. Data is collected but protected by data-shifting the study records
  - Other attributes associated with Pregnancy History are removed for data minimization
- Mortality data is available through vital records offices. The mortality data will include:
  - Pseudonymized name
  - Pseudonymized identifier
  - Date of death is needed relative to incident and treatment dates. Data is collected but protected by data-shifting the study records

>   - Cause of death

Reviewer: Either add a row for residence generalization aligned with the residence identified by the health record, or update in the earlier discussion relating to mortality data:
residence generalization aligned with the residence identified by the health record

Author: added a row: residence generalization aligned with the residence identified by the health record

>   - All other mortality data is omitted for data-minimization

Author: ```suggestion
  - Residence generalization aligned with the residence identified by the health record.
  - All other mortality data is omitted for data-minimization
```

### Determine de-identification goals

#### General goals
- Prevent Identification: remove or transform DIs and reduce QI combinability.
- Control Risk: meet non-public average thresholds with outlier protections.
- Preserve Utility: maintain longitudinal and outcomes analyses for permitted research.

#### Determine specific goals
- Entities: patients (primary), providers/facilities (as needed to avoid indirect identification).
- All unspecified IPS data elements are removed for data minimization.
- Identifiability target: Irreversibly Pseudonymized Data (see [Concepts](concepts.html#Identifiability)).
- Risk threshold: Non-public average risk ≤ 0.075, with outlier protections (see [Process → Determine goals](process.html)).
- Rationale: controlled sharing via permits; medium attack possibility and impact; sensitive health data.
- Element-level specification (Data Elements)

Data Types: The IPS format requested contains primarily structured data with some attributes containing textual data content. There are no Medical imaging data, Bio-signal data, Genetic data, Textual data, or Multi-modal data. Available to this research study through the IPS structured format.

> This implements the element-by-element de-identification design as defined in [Process](process.html). The `Identifier Type` (DI/QI/NI) classification and `Handling/Notes` capture the selected transformations for each data element.

Reviewer: Add to end: available in the source IPS and VRDR standardized content.

Author: done

> This implements the element-by-element de-identification design as defined in [Process](process.html). The `Identifier Type` (DI/QI/NI) classification and `Handling/Notes` capture the selected transformations for each data element.

Author: ```suggestion
This implements the element-by-element de-identification design as defined in [Process](process.html). The `Identifier Type` (DI/QI/NI) classification and `Handling/Notes` capture the selected transformations for each data element available in the source IPS and VRDR standardized content..
```

  {:.grid}
- Stage 3 (Recipient Verification): recipient-side verification of risk level and constraints; use enclave or controlled environments when appropriate.

See [Process → Multi-stage design](process.html).

### Governance & Controls

- Access control: least privilege; roles for policy, execution, supervision.
- Secrets management: salts, random seeds, and mapping tables stored in secure vaults; limited access.
- Secure transfer + encryption: SFTP/HTTPS; AES-256 at rest; auditable provenance.
- Documentation: retain element-level specification, scripts/config, validation reports (see [Process → Governance](process.html)).

#### Process Identifiers
Batch IDs, transformation logs, and audit trails generated during de-identification are maintained and protected. Mapping tables and seeds are segregated with strict access controls.

### Outcome & Classification

Released dataset classification: Irreversibly Pseudonymized Data for non-public controlled sharing, meeting average risk ≤ 0.075 with safeguards. Residual risks are managed via suppression/generalization and contractual controls; utility confirmed for population metrics and longitudinal trends.

### FHIR Example

> #### IPS Data Element Mappings (FHIR)

Reviewer: Add VRDR part of the example for Date of Death, Cause of Death, Patient name, address, occupation and industry. all other attributes may be omitted for minimization

Should use https://hl7.org/fhir/us/vrdr/StructureDefinition-vrdr-death-certificate.html (composition)
Decedent: https://hl7.org/fhir/us/vrdr/StructureDefinition-vrdr-decedent.html (for Patient name, address, death indicator, and death date)
Usual Work (occupation and industry): (https://hl7.org/fhir/us/vrdr/StructureDefinition-vrdr-decedent-usual-work.html),  using values:
4221 | Travel agency and related clerks OID:1.3.6.1.4.1.19376.1.5.3.1.3.43.48.3  for the occupation and 
5110 Passenger air   transport , OID:1.3.6.1.4.1.19376.1.5.3.1.3.43.48.2 for the industry
and 
Automated Underlying Cause of death
https://hl7.org/fhir/us/vrdr/StructureDefinition-vrdr-automated-underlying-cause-of-death.html using value: ICD-10: U07. 1 Covid-19

Author: added a separate section "FHIR VRDR Example (Death Certificate Document)"
